mappings/net/minecraft/world/World.mapping

```diff
@@ -11,7 +11,7 @@ CLASS none/aiw net/minecraft/world/World
 	FIELD B lootTableHandler Lnone/bbm;
 	FIELD C profiler Lnone/os;
 	FIELD D scoreboard Lnone/bdb;
-	FIELD E isRemote Z
+	FIELD E isProxy Z
```

Comment:
Because it says whether the world is a remote proxy. IMHO, isRemote is more descriptive than isProxy.